src/condor_utils/param_info.in

```diff
@@ -6109,6 +6109,96 @@ default : @end
 	IS_OWNER=(START =?= False)
 @end
 
+[$FEATURE.POLICY_HOURS_TO_START]
+description=expression fragments for running jobs only certain hours or days
+default : @end
+    # Desktops are never idle during the Workday, use 24 hour day, fractional hours
+    WorkdayBeginHour=$(1:8)
+    WorkdayEndHour=$(2:12+5)
+    # don't start jobs with 30 minutes of the start of day
+    WorkdayBeginMinuteStartJobs=($(WorkdayBeginHour))*60 - 29.5
+    # begin evicting jobs 10 minutes before the start of the day
+    WorkdayBeginMinuteStopJobs=($(WorkdayBeginHour))*60 - 9.5
+    WorkdayEndMinute=($(WorkdayEndHour))*60 + 0.5
+    RunJobsHours=ClockMin < $INT(WorkdayBeginMinuteStartJobs) || ClockMin > $INT(WorkdayEndMinute)
+    NoJobsHours=ClockMin > $INT(WorkdayBeginMinuteStopJobs) && ClockMin < $INT(WorkdayEndMinute)
+    if $(3?)
+       WorkweekFirstDay=$(3:1)
+       WorkweekLastDay=$(4:5)
+       IsWeekend=ClockDay < $INT(WorkweekFirstDay) || ClockDay > $INT(WorkWeekLastDay)
+       IsWorkday=ClockDay >= $INT(WorkweekFirstDay) && ClockDay <= $INT(WorkWeekLastDay)
+       RunJobsHours=$(IsWeekend) || $(RunJobsHours)
+       NoJobsHours=$(IsWorkday) && $(NoJobsHours)
+    endif
+    RunJobsHours=($(RunJobsHours))
+    NoJobsHours=($(NoJobsHours))
+@end
+
+[$POLICY.DESKTOP_IDLE]
```

Comment:
Do we need the regular desktop policy fragments for this policy?

Reply:
the pure hours policy should not load the policy fragments.